src/commands/app/run.js

```diff
@@ -35,12 +37,57 @@ class Run extends BaseCommand {
       fs.ensureDirSync(path.dirname(PRIVATE_KEY_PATH))
       // if they do not exists, attempt to create them
       if (!fs.existsSync(PRIVATE_KEY_PATH) && !fs.existsSync(PUB_CERT_PATH)) {
-        // todo: store them in global config when we generate them, so we don't need
-        // to repeatedly accept them
-        const CertCmd = this.config.findCommand('certificate:generate')
-        if (CertCmd) {
-          const Instance = CertCmd.load()
-          await Instance.run([`--keyout=${PRIVATE_KEY_PATH}`, `--out=${PUB_CERT_PATH}`, `-n=${this.appName}.cert`])
+        // 1. do they exist in global config?
+        const devConfig = coreConfig.get('aio-dev.dev-keys')
+        if (devConfig) {
+          // yes? write them to file
+          fs.writeFile(PRIVATE_KEY_PATH, devConfig.privateKey)
+          fs.writeFile(PUB_CERT_PATH, devConfig.publicCert)
+        } else {
+          // 2. generate them
+          const CertCmd = this.config.findCommand('certificate:generate')
+          if (CertCmd) {
+            console.log('generating certificate')
+            const Instance = CertCmd.load()
+            await Instance.run([`--keyout=${PRIVATE_KEY_PATH}`, `--out=${PUB_CERT_PATH}`, '-n=DeveloperSelfSigned.cert'])
+          } else {
+            // could not find the cert command, not sure what we should do here
```

Comment:
Maybe we can throw error saying - error while generating certificate - no certificate:generate command found